scripts/zones/Aht_Urhgan_Whitegate/TextIDs.lua

```diff
@@ -16,11 +16,12 @@ ITEM_CANNOT_BE_OBTAINEDX = 221; -- You cannot obtain the ?Possible Special Code:
 SANCTION = 9799; -- You have received the Empire's Sanction.
 
 -- Quest dialogs
- ZASSHAL_DIALOG = 10993; -- 'ang about. Looks like the permit you got was the last one I 'ad, so it might take me a bit o' time to scrounge up some more. 'ere, don't gimme that look. I'll be restocked before you know it.
-MUSHAYRA_DIALOG = 4962; -- Sorry for all the trouble. Please ignore Hadahda the next time he asks you to do something.
- HADAHDA_DIALOG = 4913; -- Hey, think you could help me out?
- PAY_DIVINATION = 8765; -- You pay 1000 gil for the divination.
-
+ ZASSHAL_DIALOG         = 10993; -- 'ang about. Looks like the permit you got was the last one I 'ad, so it might take me a bit o' time to scrounge up some more. 'ere, don't gimme that look. I'll be restocked before you know it.
+MUSHAYRA_DIALOG         = 4962; -- Sorry for all the trouble. Please ignore Hadahda the next time he asks you to do something.
+ HADAHDA_DIALOG         = 4913; -- Hey, think you could help me out?
+ PAY_DIVINATION         = 8765; -- You pay 1000 gil for the divination.
+ YOU_CAN_BECOME_PUP     = 5831; -- You can now become a puppetmaster!
```

Comment:
Note to whomever eventually merges this.  This 5831 text id is pre- November 2017 version shift and won't be covered by DragonArmor's open PR

Reply:
November update merged and I added this textID in so thsi PR won't need to. Rebase leaving this file out of the PR and yer good.